apps/asap-server/src/routes/research-outputs.route.ts

```diff
@@ -49,6 +51,33 @@ export const researchOutputRouteFactory = (
     },
   );
 
+  researchOutputRoutes.post('/research-outputs', async (req, res) => {
+    const {
```

Comment:
surely we would like to add some validation here, no?

Reply:
Doing this now, we had this but can't find it :)

Reply:
Done